src/js/module/component/button.js

```diff
@@ -36,4 +36,8 @@ export default View.extend({
         .html(view)
         .toggleClass('hidden', !this.model.get('visible'));
   },
+
+  proxyEvent(event) {
```

Comment:
What's the reason of this proxy? Is it that we need to define all callbacks in `events` attribute? Because it looks for me that
`
this.submit.on('click', this.onClickSubmit, this);
`
should work directly?